bin/build

```diff
@@ -42,11 +42,18 @@ else
     cp -r "${MW_LAYER}/node_modules/." "${NODE_MODULES_DIR}"
 fi
 
+# Setup user function as a node module dependency in the middleware and allow
+# the user function to be required by name.
+USER_FUNCTION_PACKAGE_NAME=$(jq -r '.name' < /workspace/package.json)
+echo -n "$USER_FUNCTION_PACKAGE_NAME" > "$MW_LAYER/env/USER_FUNCTION_PACKAGE_NAME.override"
+ln -s /workspace "$MW_LAYER/node_modules/$USER_FUNCTION_PACKAGE_NAME"
+
 mkdir -p "$MW_LAYER/env.launch"
 if [[ ! -z "${DEBUG_PORT}" ]]; then
   echo -n "--inspect=0.0.0.0:${DEBUG_PORT}" > "$MW_LAYER/env.launch/NODE_OPTIONS.override"
 fi
 echo "launch = true" > "$MW_LAYER.toml"
 
+
 # test that we can load the user function
-echo "require('$MW_LAYER/dist/userFnLoader').default()" | node
+echo "require('$MW_LAYER/dist/userFnLoader').default()" | USER_FUNCTION_PACKAGE_NAME=$USER_FUNCTION_PACKAGE_NAME node
```

Comment:
do we need to prefix this env var with something like `SF_` (mostly to make it clear where it comes from)?

Reply:
My intent was to keep consistency with `USER_FUNCTION_URI` that Riff uses. I don't mind adding a prefix. Or replacing `USER` with `SF`.

Reply:
I don't have a strong preference, but I also don't feel like we need to align with the riff pattern. so the `SF_` looks good to me, and probably future proofs us